### PR TITLE
added readme.md file

### LINKED ISSUE #1: Readme.md
- Add instructions on how to setup the project locally

### PR DESCRIPTION
**Pull request by ImAnshuJoshi**
Readme.md file added .

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- MongoDB
+
+And of course it is itself open source with a public repository
+on GitHub.
+
+## Installation
+
+- Make sure you have NodeJS 16.0+ installed lcoally on your device
+- Create a .env file in the root directory of the project and copy the contents of .env.example into it
+- Create a .env file in the root directory of the project and copy the contents of .env.example into it.
```
Comment (sohan2410) [inline readme.md]: Duplicate line, please delete it!

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+```
+npm i
+```
+
+- Start the application using the following command
+
+```
+npm start
+```
+
```
Comment (sohan2410) [inline readme.md:35]: Can you please write the instructions to run in development and production mode.

**Comment by sohan2410** [comment]
This looks great @ImAnshuJoshi ! I just left a couple of minor comments, we can merge this once they're fixed.

**Comment by ImAnshuJoshi** [comment]
@sohan2410 , sure 
Working on it.

**Comment by sohan2410** [comment]
@ImAnshuJoshi please add description in PR

**Comment by ImAnshuJoshi** [comment]
@sohan2410  , i have made the changes according to the requirements .

**Pull request merged**